src/Domain/Creators/AlignedLattice.hpp

```diff
@@ -131,10 +131,17 @@ class AlignedLattice : public DomainCreator<VolumeDim> {
         "Initial number of grid points in each dimension."};
   };
 
+  struct RefinedRefinement {
```

Comment:
how about just `Refinement`

Reply:
I don't like "Refinement" paired with "InitialRefinement", but how about "Refinement" and "DefaultRefinement" (and similar for GridPoints)?

Reply:
sounds good to me